modular_skyrat/modules/customization/modules/mob/dead/new_player/body_markings/body_markings.dm

```diff
@@ -66,6 +66,18 @@
 	icon_state = "flushed_cheeks"
 	default_color = "#CCCCCC"
 	affected_bodyparts = HEAD
+	
+/datum/body_marking/other/blank_face
```

Comment:
Undocumented addition. Please document all your changes.

Reply:
#17445 See here. It has already been merged as far as I know. Sorry about that.

Reply:
I see the problem. Should be good now.